cli/pkg/kctrl/cmd/core/config_factory.go

```diff
@@ -123,6 +131,11 @@ func (f *ConfigFactoryImpl) clientConfig() (bool, clientcmd.ClientConfig, error)
 	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
 	overrides := &clientcmd.ConfigOverrides{}
 
+	if len(path) == 0 && len(context) == 0 && f.defaultKubeconfigOverrideContext != "" && f.defaultKubeconfigOverridePath != "" {
```

Comment:
No strong opinions but do we want separate these conditions?

Reply:
Maybe just throw in a new line?

Reply:
I think it's fine this way, I was just thinking if we want to keep those conditions separate.